kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/service/GwtSessionService.java

```diff
@@ -31,5 +31,6 @@ public interface GwtSessionService extends RemoteService {
 
     public GwtUserConfig getUserConfig(GwtXSRFToken token) throws GwtKuraException;
 
-    public void updatePassword(final String newPassword) throws GwtKuraException;
+    public void updatePassword(GwtXSRFToken token, final String oldPassword, final String newPassword)
```

Comment:
Please add an audit annotation to this method

Reply:
Done.